rotkehlchen/data/importer.py

```diff
@@ -1014,3 +1015,172 @@ def import_shapeshift_trades_csv(self, filepath: Path) -> Tuple[bool, str]:
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    def _consume_uphold_transaction(self, csv_row: Dict[str, Any]) -> None:
+        """
+        Consume the file containing both trades and transactions from uphold.
+        This method can raise:
+        - UnknownAsset
+        - DeserializationError
+        - KeyError
+        """
+        timestamp = deserialize_timestamp_from_date(
+            date=csv_row['Date'],
+            formatstr='%a %b %d %Y %H:%M:%S %Z%z',
+            location='uphold',
+        )
+        destination = csv_row['Destination']
+        destination_asset = asset_from_uphold(csv_row['Destination Currency'])
+        destination_amount = deserialize_asset_amount(csv_row['Destination Amount'])
+        origin = csv_row['Origin']
+        origin_asset = asset_from_uphold(csv_row['Origin Currency'])
+        origin_amount = deserialize_asset_amount(csv_row['Origin Amount'])
+        if csv_row['Fee Amount'] == '':
+            fee = FVal(ZERO)
```

Comment:
```suggestion
            fee = ZERO
```

`ZERO` is already an FVal